Formula/qthreads.rb

```diff
@@ -18,16 +13,12 @@ class Qthreads < Formula
     sha256 cellar: :any, mojave:   "0eb14aec995c438dab677dc3084bf772a5ff083812a5d3a0eac988ed08496a42"
   end
 
-  head do
-    url "https://github.com/Qthreads/qthreads.git", branch: "main"
-
-    depends_on "autoconf" => :build
-    depends_on "automake" => :build
-    depends_on "libtool" => :build
-  end
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
 
   def install
-    system "./autogen.sh" if build.head?
+    system "./autogen.sh"
```

Comment:
Does their `configure` script not work?

Reply:
It doesn't exist anymore - they removed it in this release so now running `autogen.sh` is mandatory.